<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>车间生产统计工具</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>选择工序</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox_process">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>工序：水刺 | 编码：10</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>工序：脱漂烘干 | 编码：20</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>工序：检布（验布） | 编码：30</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>工序：分切 | 编码：40</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>工序：分切覆膜 | 编码：50</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>一物一码</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_qrcode_id"/>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>赋码时间范围</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyLineEdit" name="lineEdit_time_cr_before"/>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>-</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MyLineEdit" name="lineEdit_time_cr_after"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_search">
        <property name="text">
         <string>查询</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_tongji">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>统计</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTableView" name="tableView">
      <property name="gridStyle">
       <enum>Qt::DashDotLine</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1004</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="actiontuichu"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="statusTip">
    <string/>
   </property>
  </widget>
  <action name="actiontuichu">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">Custom</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
